packages/turbo-types/src/types/config-v1.ts

```diff
@@ -0,0 +1,301 @@
+  /**
+   * Indicates if signature verification is enabled for requests to the remote cache. When
+   * `true`, Turborepo will sign every uploaded artifact using the value of the environment
+   * variable `TURBO_REMOTE_CACHE_SIGNATURE_KEY`. Turborepo will reject any downloaded artifacts
+   * that have an invalid signature or are missing a signature.
+   *
+   * @defaultValue `false`
+   */
+  signature?: boolean;
+
+  /**
+   * Indicates if the remote cache is enabled. When `false`, Turborepo will disable
+   * all remote cache operations, even if the repo has a valid token. If true, remote caching
+   * is enabled, but still requires the user to login and link their repo to a remote cache.
+   * Documentation: https://turbo.build/repo/docs/core-concepts/remote-caching
+   *
+   * @defaultValue `true`
+   */
+  enabled?: boolean;
+}
+
+export interface RootSchemaV1 extends BaseSchemaV1 {
+  /**
+   * A list of globs to include in the set of implicit global hash dependencies.
+   *
+   * The contents of these files will be included in the global hashing
+   * algorithm and affect the hashes of all tasks.
+   *
+   * This is useful for busting the cache based on:
+   *
+   * - .env files (not in Git)
+   *
+   * - any root level file that impacts package tasks
+   * that are not represented in the traditional dependency graph
+   * (e.g. a root tsconfig.json, jest.config.js, .eslintrc, etc.)
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globaldependencies
+   *
+   * @defaultValue `[]`
+   */
+  globalDependencies?: Array<string>;
+
+  /**
+   * A list of environment variables for implicit global hash dependencies.
+   *
+   * The variables included in this list will affect all task hashes.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globalenv
+   *
+   * @defaultValue `[]`
+   */
+  globalEnv?: Array<EnvWildcardV1>;
+
+  /**
+   * An allowlist of environment variables that should be made to all tasks, but
+   * should not contribute to the task's cache key, e.g. `AWS_SECRET_KEY`.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globalpassthroughenv
+   *
+   * @defaultValue `null`
+   */
+  globalPassThroughEnv?: null | Array<EnvWildcardV1>;
+
+  /**
+   * A priority-ordered (most-significant to least-significant) array of project-anchored
+   * Unix-style paths to `.env` files to include in the global hash.
+   *
+   * @defaultValue `null`
+   */
+  globalDotEnv?: null | Array<AnchoredUnixPath>;
+
+  /**
+   * Configuration options that control how turbo interfaces with the remote cache.
+   *
+   * Documentation: https://turbo.build/repo/docs/core-concepts/remote-caching
+   *
+   * @defaultValue `{}`
+   */
+  remoteCache?: RemoteCacheV1;
+
+  /**
+   * Enable use of the UI for `turbo`.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#ui
+   *
+   * @defaultValue `"stream"`
+   */
+  ui?: UIV1;
```

Comment:
We didn't properly capture what this field looked like in v1. It was a bool named `experimentalUI`. [source](https://turbo.build/schema.v1.json)
```suggestion
  /**
   * Enable use of the new UI for `turbo`
   *
   * Documentation: https://turbo.build/repo/docs/reference/configuration#ui
   *
   * @defaultValue `"false"`
   */
  experimentalUI?: bool;
```

Reply:
updated in 90e10024e220385151eefb6ed94c94138ffc4d53